plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -18,7 +18,7 @@ class BaseQueueConf:
     # number of gpus to use on each node
     gpus_per_node: Optional[int] = None
     # number of tasks to spawn on each node
-    tasks_per_node: int = 1
+    tasks_per_node: Optional[int] = None
```

Comment:
Maybe we should leave the default value as 1 to keep backward compatibility with older versions of the submitit launcher plugin?